### ISSUE TITLE
Incompatible with IntelliJ IDEA 2020.1 EAP

### ISSUE THREAD

**Issue by RakSrinaNa** [OpherV/gitflow4idea#276]
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request
  - [ ] puppy => You're not submitting a puppy. I already have one and he's [adorable](https://twitter.com/Opherv/status/974341249099542528)

* **What is the current behavior?**
Incompatible with IntelliJ IDEA 2020.1 EAP

* **Please tell me about your environment:**

```
IntelliJ IDEA 2020.1 EAP (Ultimate Edition)
Build #IU-201.4515.24, built on January 30, 2020
Runtime version: 11.0.5+9-b674.2 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

**Comment by CC007** [comment]
@OpherV when do you think you will have time to make the plugin compatible with 2020.1 EAP?

**Comment by OpherV** [comment]
@CC007 not sure yet, this whole Coronavirus global pandemic thing threw my life out of whack.

I'll definitely get to it the soonest I can, and I hoping it'll be sooner rather than later

**Comment by roelvs** [comment]
This new Intellij version seems to be functional with the current code (tested with PyCharm 2020.1, Build #PY-201.6668.115) if the plugin.xml is updated to reflect new version.

Temporary workaround (without rebuilding the source) until a new version is released by @OpherV :  
- Edit the plugin.xml file in the plugin zip file: gitflowidea-0.7.1.zip\gitflow4idea\lib\gitflow4idea-0.7.1.jar\META-INF\plugin.xml (I used 7-zip to update the zip/jar file without unpacking it) 
- Update the 'until-build' variable manually:  ```<idea-version since-build="182.0" until-build="201.*"/>```
- Update the archive
- Install the plugin manually (settings->plugins->install plugin from disk)

**Comment by UkonnRa** [comment]
+1, want to update to the idea 2020.1

**Comment by bill-thedbaguy** [comment]
@roelvs I followed you steps above and Git Flow Integration appears to be working on WebStorm 2020.1.
@OpherV +1 for you plugin.

**Comment by zhairuihao** [comment]
> This new Intellij version seems to be functional with the current code (tested with PyCharm 2020.1, Build #PY-201.6668.115) if the plugin.xml is updated to reflect new version.
> 
> Temporary workaround (without rebuilding the source) until a new version is released by @OpherV :
> 
> * Edit the plugin.xml file in the plugin zip file: gitflowidea-0.7.1.zip\gitflow4idea\lib\gitflow4idea-0.7.1.jar\META-INF\plugin.xml (I used 7-zip to update the zip/jar file without unpacking it)
> * Update the 'until-build' variable manually:  `<idea-version since-build="182.0" until-build="201.*"/>`
> * Update the archive
> * Install the plugin manually (settings->plugins->install plugin from disk)

Init Repo not working.
`  Error: Please have a look at the Version Control console for more details `

**Comment by fabmars** [comment]
PR submitted

**Comment by OpherV** [comment]
Fixed in 0.7.2, out now
